wazo_agid/modules/convert_pre_dial_handler.py

```diff
@@ -1,36 +0,0 @@
-# Copyright 2021-2022 The Wazo Authors  (see the AUTHORS file)
-# SPDX-License-Identifier: GPL-3.0-or-later
-
-import re
-
-from wazo_agid import agid
-from wazo_agid.dialplan_variables import CALL_OPTIONS
-
-B_REGEX = re.compile(r'b\(([\-_0-9A-Za-z]+)\^?.*?\)')
-
-
-# This AGI was added in 21.01 to avoid breaking user written dialplan
-# This AGI and all of it's calls should be deleted in a reasonable amount of time 22.01?
-def convert_pre_dial_handler(agi, cursor, args):
-    call_options = agi.get_variable(CALL_OPTIONS)
```

Comment:
Great to remove it, but I think it is still used in some places, so maybe just check to remove them there too. Ex: https://github.com/wazo-platform/xivo-config/blob/bdea51cc19abe6ac4d13f6aa13823f2d666158d1/dialplan/asterisk/extensions_lib_pre_dial.conf#L16

Reply:
According to Pawscal, this one is actually the only usage of `b` option that we want(which executes all other pre-dial handlers). 
Instead of simply removing the handler though, I will convert the converter to a "check, warn and ignore" thing instead, so if there still exists custom config using b option, it will be ignored with a warning instead of just breaking things.